.goreleaser.yaml

```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
+      - "--platform=linux/amd64"
```

Comment:
The [docs](https://goreleaser.com/cookbooks/multi-platform-docker-images/) also include `--pull` in the build flags. Maybe to ensure you use layer caches if they are available?

Reply:
Will add - I must have missed that in the original Docker build.